### ISSUE TITLE
Imput device

### ISSUE THREAD

**Issue by thedoggaming16** [helge17/tuxguitar#605]
Hello, about the tuner, I'm asking if it's possible to change the ''Imput device'' and if yes, how to change it. Thanks ! (I'm on version 1.6.3)

**Comment by guiv42** [comment]
I don't think it's possible to change the input device used by the tuner. What are you exactly looking for? Do you have several microphones on your machine?
I would say that it's the role of the operating system to configure the different possible audio inputs, and to decide which microphone shall be used.

**Comment by thedoggaming16** [comment]
Yes I have several microphones, one for my voice obviously but I also have a IRig 2.0 that use a different Input and I use it for recording my guitar.

**Comment by guiv42** [comment]
OK, understood. No idea if it is feasible or not... might be very OS-dependent.
